test/files/pos/t6047.scala

```diff
@@ -1,3 +1,4 @@
+/* scalac: -language:experimental.macros */
```

Comment:
cleverly, you can also just import it.

Reply:
Yes, but I wanted some things not in `neg` that would fail if it didn't work. Most just check that it works despite the flags.